packages/replay/src/coreHandlers/handleDom.ts

```diff
@@ -47,13 +48,31 @@ function handleDom(handlerData: DomHandlerData): Breadcrumb | null {
     return null;
   }
 
+  // `__sn` property is the serialized node created by rrweb
+  const serializedNode =
+    targetNode && '__sn' in targetNode && targetNode.__sn.type === NodeType.Element ? targetNode.__sn : null;
+
   return createBreadcrumb({
     category: `ui.${handlerData.name}`,
     message: target,
     data: {
-      // Not sure why this errors, Node should be correct (Argument of type 'Node' is not assignable to parameter of type 'INode')
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      ...(targetNode ? { nodeId: record.mirror.getId(targetNode as any) } : {}),
+      ...(serializedNode
+        ? {
+            nodeId: serializedNode.id,
+            node: {
+              id: serializedNode.id,
+              tagName: serializedNode.tagName,
+              textContent: targetNode
+                ? Array.from(targetNode.childNodes)
+                    .filter((node: Node | INode) => '__sn' in node && node.__sn.type === NodeType.Text)
+                    .map(node => node.textContent)
+                    .join('')
+                : '',
+              // TODO: strict list of attributes
```

Comment:
Is this TODO needed here? As there is already a strict list implemented here? :)